app/admin/topics.rb

```diff
@@ -0,0 +1,17 @@
+ActiveAdmin.register Topic do
+  permit_params :title
+  # See permitted parameters documentation:
+  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+  #
+  # Uncomment all parameters which should be permitted for assignment
+  #
+  # permit_params :title
+  #
+  # or
+  #
+  # permit_params do
+  #   permitted = [:title]
+  #   permitted << :other if params[:action] == 'create' && current_user.admin?
+  #   permitted
+  # end
```

Comment:
Why the comment code?
Also, is better if you specify which attributes you want to show so you can have more control.